isb_lib/core.py

```diff
@@ -557,6 +565,7 @@ def __init__(
         status: int = 200,
         page_size: int = 5000,
         offset: int = 0,
+        limit: int = -1,
```

Comment:
Added a `limit` parameter to the Thing record iterator -- it was nice to be able to just test the solr ingest when fetching 5000 Things.